### PR TITLE
Functionality to disable specific lyrics provider 

### LINKED ISSUE #53: Language Mismatch when no lyrics are found.
system language is "en" but when no lyrics are found it writes in wrong language. other then the one specified 
example 
image

### PR DESCRIPTION
**Pull request by AG4lyf**
kinda closes #53

### CODE REVIEW

**Comment by NyaomiDEV** [comment]
A couple points to this:
- Please respect my code formatting. I am using camel case and not snake case. Also there are missing spaces around tokens such as `:` and `=`;
- Your description in the configuration file is rushed; please use proper English and formatting also there;
- `src/main/integrations/lyrics.ts`: Imports must not specify `.js` at the end, as per code style choice;
- `src/main/integrations/lyrics.ts`: Please assign `Config` as close as possible to the checks; there's no reason to get it before as it's not used elsewhere in the function body;
- `src/main/integrations/lyrics.ts`: The use of `Object.assign` is not proper, meaning that you did not test this code. You should assign an object containing the provider if you want to go that route; however, a simpler way to do this could just be to use regular assignations:
  ```ts
		// Line 26 onwards
		let providers: any = {}; // marking providers as 'any' avoids TS issues
		if(Config.lyricsProvider.Musixmatch) // accounting for the first suggestion
			providers.Musixmatch = Musixmatch;
		
		if(Config.lyricsProvider.NetEase)
			providers.NetEase = NetEase;
		// and so on...
  ```

The PR is not mergeable in this state; please fix all those issues and then I will review it once more.

**Comment by AG4lyf** [comment]
`Imports must not specify .js at the end, as per code style choice;` 
I can see other files using this like `seekbar.ts, screen.ts, nowplaying.ts, lyrics.ts, event.ts, buttons.ts, appicon.ts` under  `src/www/lib`

File: src/main/integrations/lyrics.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { query as NetEase } from "../lyricproviders/netease";
 import { query as Genius } from "../lyricproviders/genius";
 import { query as MetadataQuery } from "../lyricproviders/metadata";
 import type { Lyrics, Metadata } from "../../types";
+import {getAll as get_config} from "../config";
```
Comment (NyaomiDEV) [inline src/main/integrations/lyrics.ts:9]: Missing space around the brackets; `get_config` should be `getConfig`.

File: src/types.ts [hunk 2]
```diff
@@ -75,6 +76,13 @@ export type SpotifyConfig = {
 	clientSecret: string
 }
 
+export type LyricsProviderConfig = {
```
Comment (NyaomiDEV) [inline src/types.ts:79]: Please properly capitalize `Genius`, `Musixmatch`, `NetEase`, `Metadata`; and change accordingly in other files.

Also reorder them in order of importance (as they are used):
- Musixmatch goes first
- NetEase second
- Genius
- Metadata

Do so in the configuration file too.

**Comment by NyaomiDEV** [comment]
> seekbar.ts, screen.ts, nowplaying.ts, lyrics.ts, event.ts, buttons.ts, appicon.ts

The `lib` parts compiles into ES6 for use in browsers; what you're modifying compiles into CommonJS for use in the main Electron context.

Please take a look at the review I started.

File: assets/config.json5 [hunk 3]
```diff
@@ -26,6 +26,13 @@
 	// Specify your Musixmatch User Token.
 	// Sunamu will automatically generate it if you don't have your own.
 	mxmusertoken: "",
+	// Are you not liking lyrics from a specific source? You can disable it here.
+	lyricsProvider: {
```
Comment (NyaomiDEV) [inline assets/config.json5:30]: Please modify capitalization and order as outlined in `types.ts`.

File: assets/config.json5 [hunk 4]
```diff
@@ -26,6 +26,13 @@
 	// Specify your Musixmatch User Token.
 	// Sunamu will automatically generate it if you don't have your own.
 	mxmusertoken: "",
+	// Are you not liking lyrics from a specific source? You can disable it here.
```
Comment (NyaomiDEV) [inline assets/config.json5:29]: Please paste the following here:
```json5
// Sunamu makes use of online and offline services to get lyrics.
// Their priority order is outlined here, top to bottom.
// If you're having problems with some services, you can disable them here.
```

**Review by NyaomiDEV — CHANGES_REQUESTED** [review]

File: assets/config.json5 (hunk 3 above)
Comment (AG4lyf) [reply assets/config.json5]: I am confused by the term "capitalization" 
isn't this a proper camelCase ?

File: src/main/integrations/lyrics.ts (hunk 1 above)
Comment (NyaomiDEV) [reply src/main/integrations/lyrics.ts]: get_config should be getConfig.

File: assets/config.json5 (hunk 3 above)
Comment (NyaomiDEV) [reply assets/config.json5]: Capitalization I meant for the services. I hate lowercase.

File: src/main/integrations/lyrics.ts [hunk 5]
```diff
@@ -6,6 +6,7 @@ import { query as NetEase } from "../lyricproviders/netease";
 import { query as Genius } from "../lyricproviders/genius";
 import { query as MetadataQuery } from "../lyricproviders/metadata";
 import type { Lyrics, Metadata } from "../../types";
+import { getAll as get_config } from "../config";
```
Comment (NyaomiDEV) [inline src/main/integrations/lyrics.ts:9]: You're just missing `get_config` => `getConfig` here!

**Review by NyaomiDEV — CHANGES_REQUESTED** [review]

**Comment by AG4lyf** [comment]
me using snakecases as default make sense. cuz I come from Python Background. and we only use snakecase there.

**Review by NyaomiDEV — APPROVED** [review]

**Comment by NyaomiDEV** [comment]
I want to know if this PR works as advertised in this state...

**Comment by AG4lyf** [comment]
it does. tested and worked. 
There can be a little more addition. 
like removing the huge "No Lyrics Avaialable" its just something that annoys me.

**Comment by NyaomiDEV** [comment]
> it does. tested and worked. There can be a little more addition. like removing the huge "No Lyrics Avaialable" its just something that annoys me.

WHY? It is meant to inform the user that lyrics aren't available!

**Comment by NyaomiDEV** [comment]
So, IDK if that was my fault or not, but the metadata query code was faulty lol.

Anyway, I fixed it now, and also the list now reorders the services' priority.

**Pull request merged**